src/it/projects/complete-multimodule-parent-pom-cifriendly/pom.xml

```diff
@@ -15,7 +15,7 @@
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>flatten-maven-plugin</artifactId>
 				<configuration>
-					<flattenMode>resolveCiFriendliesOnly</flattenMode>
+					<flattenMode>version</flattenMode>
```

Comment:
Please do not missuse existing ITs to test a new feature. You would need to create a new integration test instead.

Reply:
IMHO we should either "fix" the existing mode, or if we keep it create a new IT for `version` mode.

Reply:
As per https://github.com/mojohaus/flatten-maven-plugin/pull/114#issuecomment-566969124 `resolveCiFriendlies` should be fixed. :)